public/app/features/dashboard/components/PanelEditor/PanelOptionsTab.tsx

```diff
@@ -152,5 +222,56 @@ export const PanelOptionsTab: FC<Props> = ({
     </OptionsGroup>
   );
 
+  if (config.featureToggles.panelLibrary) {
+    elements.push(
```

Comment:
![image](https://user-images.githubusercontent.com/2376619/106439100-98a88980-6477-11eb-8cae-420d52cc3d40.png)

I think going forward we could optimize the card's display depending on the available space in the options pane. Maybe show less information when pane is not wide enough.

I'm also not sure about the information being shown on the card (at least when adding a new panel to a dashboard). I think the information bout how many dashboards the panel is used is probably relevant when the user wants to remove the panel. In other cases, it feels like it may be adding unnecessary noise. WDYT @jessover9000?

Reply:
yea, agree, would try to keep this card as simple as possible to start with (star, delete, copy). 

What does the `(x) 3` mean

Reply:
Per jess's design, it means the number of template variables used by the panel. Currently just a placeholder though

Reply:
ah, ok. Let's skip that for now